wave-fe/src/scss/tailwind/custom-components.scss

```diff
@@ -0,0 +1,23 @@
+.btn {
+  @apply bg-gray-400;
+  @apply bg-blue-800;
+  @apply px-4;
+  @apply py-2;
+  @apply rounded;
+  @apply text-black;
```

Comment:
`@apply text-black` line can be removed, its being overwritten by the line below

Reply:
fixed